meet-data/raw/raw-probe

```diff
@@ -60,27 +49,5 @@ def getmeetlist(html):
 
     return urls
 
-
-def getenteredurls():
-    urls = []
-    for dirname, subdirs, files in os.walk(FEDDIR):
-        if 'URL' in files:
-            with open(dirname + os.sep + 'URL', 'r') as fd:
-                for k in fd.readlines():
-                    urls.append(k.strip())
-    return urls
-
-
-def main():
-    html = gethtml(URL)
-    meetlist = getmeetlist(html)
-
-    known = getenteredurls()
-
-    for m in meetlist:
-        if not m in known:
-            print(color('[RAW] ') + m)
-    print(color('[RAW] ') + "Continue working through archive.")
-
 if __name__ == '__main__':
-    main()
+    probe.main('[RAW]', color, getmeetlist, URL)
```

Comment:
The `main()` function should definitely remain in this file. Helper libraries shouldn't control `main`.